SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -412,6 +413,64 @@ bool SaveSchemaDatabase(const FString& PackagePath)
 	SchemaDatabase->ComponentIdToClassPath = CreateComponentIdToClassPathMap();
 	SchemaDatabase->LevelComponentIds = LevelComponentIds;
 
+	// Generate hash
+	{
+		FString DescriptorPath = FPaths::Combine(SpatialGDKServicesConstants::SpatialOSDirectory, TEXT("build/assembly/schema/schema.descriptor"));
+		IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
+
+		TUniquePtr<IFileHandle> FileHandle(PlatformFile.OpenRead(DescriptorPath.GetCharArray().GetData()));
+		if (!FileHandle)
+		{
+			FMessageDialog::Debugf(FText::FromString(TEXT("Failed to open schema.descriptor generated by the schema compiler!")));
```

Comment:
Nit: Maybe add the FullPath to this to a dev can check for issues like write access